### PR TITLE
H.E.V Armor set

### PR DESCRIPTION
**Pull request by Weisyl**
This adds the H.E.V suit and helmet from the Half-Life series.  http://puu.sh/kMvhm/eadaacf3ce.png

### CODE REVIEW

**Comment by Weisyl** [comment]
Also, credit to Nienhaus for all of the sprites.

**Comment by Weisyl** [comment]
Front: http://puu.sh/kMKYO/53e38605a2.png
Back: http://puu.sh/kMKXB/db0768562d.png
Side (facing west): http://puu.sh/kMKVQ/ebe482d4a0.png
Side (facing east): http://puu.sh/kMKWQ/e5bf3faf8e.png

**Comment by Foosili** [comment]
Damn spiffy :+1:

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -35,6 +35,29 @@
...
+	gas_transfer_coefficient = 0.01
+	permeability_coefficient = 0.01
+	flags = STOPSPRESSUREDMAGE | THICKMATERIAL
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	strip_delay = 120
```
Comment (AliiTea) [inline code/modules/clothing/suits/armor.dm:53]: Two sets of put_on and strip delay??

Comment (Weisyl) [inline code/modules/clothing/suits/armor.dm:53]: Woops, lemme fix that real quick.

**Pull request merged**